<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>www</class>
 <widget class="QWidget" name="www">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>www</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тестовая область</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButton_test">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ToolButton -1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_test">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PushButton +1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="6">
       <widget class="QSlider" name="verticalSlider_test">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_test_2">
        <property name="text">
         <string>Значение слайдеров:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_test">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_test">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_test"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_test"/>
      </item>
      <item row="5" column="1">
       <widget class="QDial" name="dial_test"/>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QSlider" name="horizontalSlider_test">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="4">
       <widget class="QCalendarWidget" name="calendarWidget_test"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_test</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_test</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_test</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_test</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_test</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_test</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_test</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>comboBox_test</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_test</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_test</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
